CHANGELOG.md

```diff
@@ -1,3 +1,147 @@
+# 1.6.10-beta01 (April 2024)
+
+_Changes since 1.6.2_
+
+## Highlights
+
+- Experimental multiplatform support of `Lifecycle` and `ViewModel`. See [documentation](https://www.jetbrains.com/help/kotlin-multiplatform-dev/compose-lifecycle.html) for details
+- Experimental multiplatform support of Jetpack Navigation. See [documentation](https://www.jetbrains.com/help/kotlin-multiplatform-dev/compose-navigation-routing.html) for details or check out the [nav_cupcake project](https://github.com/MatkovIvan/nav_cupcake) which was converted from the [Navigate between screens with Compose](https://developer.android.com/codelabs/basic-android-kotlin-compose-navigation#2) Android codelab
+- [Support multi-module projects and libraries publication with Compose resources](https://github.com/JetBrains/compose-multiplatform/pull/4454) <sub>_(Kotlin `2.0.0-Beta05` or higher is required)_</sub>
+
+## Known issues
+
+- [Crash on pre-iOS 17 devices due to loading `UITextLoupeSession` by K/N](https://github.com/JetBrains/compose-multiplatform/issues/4644)
```

Comment:
```suggestion
- [**The iOS target crashes on iOS < 17**](https://github.com/JetBrains/compose-multiplatform/issues/4644)
```
or
"Crash at startup on pre-iOS 17 devices" (but I vote for the first)

Otherwise it sounds like a local crash that sometimes happens with `UITextLoupeSession` in the console.

Reply:
I'd provide a bit more detail than just "it crashes"